tensorflow-mixin/dashboards/tensorflow-overview.libsonnet

```
@@ -681,10 +683,22 @@ local containerLogsPanel = {
             allValues='.+',
             sort=1,
           ),
+          template.new(
+            'cluster',
+            promDatasource,
+            'label_values(:tensorflow:serving:request_count{}, cluster)' % $._config,
+            label='Cluster',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.*',
+            hide=if $._config.enableMultiCluster then '' else 'variable',
+            sort=0
+          ),
           template.new(
             'instance',
             promDatasource,
-            'label_values(:tensorflow:serving:request_count{job=~"$job"}, instance)',
+            'label_values(:tensorflow:serving:request_count{%(tensorflowSelector)s}, instance)', % $._config,
```

Do not need the second last comma at the end
```suggestion
            'label_values(:tensorflow:serving:request_count{%(tensorflowSelector)s}, instance)' % $._config,
```